mrchecker-framework-modules/mrchecker-database-module/pipelines/CI/Jenkinsfile_node.groovy

```diff
@@ -26,8 +26,8 @@ https://repo.maven.apache.org/maven2/com/capgemini/ntc/
 Snapshot
 https://oss.sonatype.org/content/repositories/snapshots/com/capgemini/ntc/''', name: 'IS_TO_DEPLOY_REMOTE_NEXUS'), 
 			string(defaultValue: '', description: 'Application version number. If empty, pom.xml version will be taken', name: 'VERSION'), 
-			string(defaultValue: 'origin/develop', description: 'Optional variable. What is your "master" branch', name: 'MAIN_BRANCH'), 
-			string(defaultValue: 'https://github.com/devonfw-forge/mrchecker-source.git', description: 'Optional variable. Which repo to run', name: 'GIT_REPO'),
+			string(defaultValue: 'origin/develop', description: 'Optional variable. What is your "master" branch', name: 'MAIN_BRANCH'),
+            string(defaultValue: 'https://github.com/devonfw-forge/mrchecker-source.git', description: 'Optional variable. Which repo to run', name: 'GIT_REPO'),
```

Comment:
Inconsistent formatting, this file should be left unchanged

Reply:
Done.